src/Compilers/VisualBasic/Portable/Parser/ParserFeature.vb

```diff
@@ -4,7 +4,7 @@ Imports System.Runtime.CompilerServices
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
```

Comment:
Rather than making this internal namespace public please move both of these types to the Microsoft.VisualBasic.CodeAnalysis namespace. Please remove all extra import statements from the other source files. The binder should not reference the internal syntax tree namespace.